bootstrap4/templates/bootstrap4/inputTypes/autocomplete/autocomplete.html

```diff
@@ -1,7 +1,10 @@
 <template name="afAutocomplete_bootstrap4">
-  <div class="autocomplete dropdown">
-    <input type="text" value="{{this.label}}" {{this.visibleAtts}} />
-    <div class="dropdown-menu"></div>
-  </div>
-  <input type="hidden" value="{{this.value}}" {{this.atts}}/>
+        <input type="text" value="{{this.label}}" {{attsPlusFormControlClass this.visibleAtts}} />
+    <div class="autocomplete dropdown">
+        <div class="dropdown-menu"></div>
+        {{#if afFieldIsInvalid name=this.name}}
+            <div class="invalid-feedback">{{{afFieldMessage name=this.name}}}</div>
+        {{/if}}
```

Comment:
Oh, interesting to me because this will override the "preset" addition of the error message next to the named field, which is the hidden input in this case? We don't want the value from the visible field on submit, only the hidden input.

Reply:
The issue is that bootstrap requires the Error field to be next to the input to make it visible